arithmetization/src/main/java/net/consensys/linea/zktracer/module/ecdata/EcData.java

```diff
@@ -22,7 +22,7 @@
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 import net.consensys.linea.zktracer.ColumnHeader;
-import net.consensys.linea.zktracer.container.stacked.list.StackedList;
+import net.consensys.linea.zktracer.container.stacked.StackedList;
```

Comment:
Here I guess you just changed folder structure?

Reply:
yes, and rewrite completely the StackedList / StackedSet ;)